package.json

```diff
@@ -128,13 +130,17 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@sap_oss/odata-library": "^2.3.5",
+    "appium": "2.11.2",
+    "appium-uiautomator2-driver": "^3.7.4",
+    "appium-xcuitest-driver": "^7.24.3",
     "axios": "^1.6.0",
     "deepmerge": "^4.2.2",
     "fs-extra": "^10.0.0",
     "path": "^0.12.7",
     "pdf-parse": "^1.1.1",
     "query-string": "^7.1.0",
     "supertest": "^7.0.0",
+    "wdio-chromedriver-service": "^8.0.0",
```

Comment:
I am not sure if that should be added back. @cvakiitho I think we removed the chromedriver package for a reason, do still remember?

Reply:
The appium-uiautomator2-driver requires the wdio-chromedriver-service because it helps facilitate seamless communication between Appium and the Chrome browser when running automated tests. This service starts the Appium server in a child process, allowing you to manage ChromeDriver instances effectively, which is essential for testing web applications on Android devices via Appium with WebdriverIO (WDIO).